### ISSUE TITLE
Connection lost to bitlbee

### ISSUE THREAD

**Issue by Fogelholk** [jgeboski/bitlbee-steam#8]
Hello!
Thanks for helping me get bitlbee-steam running, now I've run into a new issue! Yay!

I created my account with the commands provided, successfully logged into Steam with your addin here and sent the authcode for logging into Steam from a new machine, so far so good!

If I check on Steam I also see that I'm logged into Mobile as well as the ordinary steam-client.
When I've been connected for a few seconds I completely lose connection to my Bitlbee-server but I'm able to reconnect instantly and this happens only after I've connected to steam with your addin.

Basically the Steam-addin tells me that it fetched the friends list and logs in, but it throws me off the Bitlbee-server when I try to see the "blist" for example.

In the end I'm unable to chat with my friends through bitlbee/irssi :(

http://pastebin.com/c8RmDShF

Is this something with your addin to do or Bitlbee itself? Running on Bitlbee 3.0.5 and Debian x86_64 as last time :)
